implementations/node/src/api/controllers/_dev/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as Router from 'koa-router';
+
```

Comment:
Only thing is I'm not sure if with that configuration a `GET /playground` will know to return `index.html` or not.